applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -268,11 +337,17 @@ static bool app_event_handler(const struct app_event_header *aeh)
 				}
 				get_req_modules(&req_modules_bm);
```

Comment:
Is the implementation correct here? Shouldn't we `get_req_modules` before the check?
```
    get_req_modules(&req_modules_bm);
    process_dvfs_states(DVFS_STATE_INITIALIZING, !module_flags_check_zero(&req_modules_bm));
```

Reply:
No it is not correct. We want to turn it on so SoC won't enter low frequency before we end up initialization.

That's why we have different order.

Reply:
You check the bitmask for zeros before initializing it. Are you sure it's what we want to do here?
I think we should actually implement the suggested improvement here.